porting/install-build/plasma-mobile.rst

```diff
@@ -0,0 +1,31 @@
+
+Plasma mobile
+=============
+
+In order to install `Plasma mobile <https://www.plasma-mobile.org>`_ you perform basically the same steps as for the :doc:`reference-rootfs`. This section only highlights the relevant differences.
+
+Install rootfs and system.img
+-----------------------------
+
+Use the plasma mobile rootfs. Download the latest version from: `<https://images.plasma-mobile.org/rootfs/>`_.
```

Comment:
There is also CAF variant (https://images.plasma-mobile.org/caf-rootfs/ ).

Reply:
added